frontend/src/core/models/articleModel.js

```diff
@@ -6,7 +6,12 @@ define(function(require) {
     urlRoot: '/api/content/article',
     _parent: 'contentObjects',
     _siblings: 'articles',
-    _children: 'blocks'
+    _children: 'blocks',
+
+    defaults: {
+      _isCollapsed: false
```

Comment:
Can you confirm this is not saved to the DB and therefore is exported on build?

Reply:
It is not saved in the DB and not exported. This is the article.json from an exported course:
```
{
    "_id": "5a7d5be53fbfedfc836c023d",
    "_type": "article",
    "_parentId": "5a7d5be53fbfedfc836c023c",
    "_onScreen": {
      "_isEnabled": false,
      "_classes": "",
      "_percentInviewVertical": 50
    },
    "_isVisible": true,
    "_isHidden": false,
    "_isAvailable": true,
    "_isOptional": false,
    "instruction": "",
    "_classes": "",
    "body": "",
    "displayTitle": "collapsed article ",
    "title": "collapsed article "
  }
```